src/dataaccess/DataAccess.jsx

```diff
@@ -36,4 +36,8 @@ export default class DataAccess {
     savePatient(patient) {

```

Comment:
I understand that it's not part of this task, but at some point the other functions should probably be updated to support sync/async as well. As a heads-up the SMART on FHIR client library makes all service calls asynchronously, so the savePatient function in that upcoming data source will have to be async.

Reply:
Agreed. I can make a JIRA to support thos additional functions. Thanks Dylan!